module/tty0tty.c

```diff
@@ -51,6 +51,11 @@ speed_t tty_termios_input_baud_rate(struct ktermios *termios);
 #endif
 
 
+#if LINUX_VERSION_CODE < KERNEL_VERSION(5, 15, 0)
```

Comment:
Please change to `LINUX_VERSION_CODE >= KERNEL_VERSION(5, 15, 0)` to work with previous versions.

Reply:
It work, but emit redefined warnings. That's what confused me. 

```
  CC [M]  /tmp/tty0tty/module/tty0tty.o
/tmp/tty0tty/module/tty0tty.c:55: warning: "tty_alloc_driver" redefined
   55 | #define tty_alloc_driver(x, y) alloc_tty_driver(x)
      | 
In file included from /usr/src/linux-headers-5.14.0-2-common/include/linux/tty.h:9,
                 from /tmp/tty0tty/module/tty0tty.c:36:
/usr/src/linux-headers-5.14.0-2-common/include/linux/tty_driver.h:339: note: this is the location of the previous definition
```